### PR TITLE
NFLOG support, ish, maybe, sort of

### PR DESCRIPTION
**Pull request by gaima8**
Due to my incompetence with git this is a refilling of my crummy NFLOG patch.

### CODE REVIEW

**Comment by sirius** [comment]
I'll try to take a look at this during the weekend, including some of the other over due PRs.

A quick question: is there a significant change in the code, or is it just a squash of the four previous commits?

Thanks for contributing!

**Comment by gaima8** [comment]
Hey Sirius,

The code is exactly the same as it was a few months ago.
I just screwed up with git.

Mike

File: src/res/configlets/linux317/automatic_rules [hunk 1]
```diff
@@ -0,0 +1,129 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/automatic_rules]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/block_action [hunk 2]
```diff
@@ -0,0 +1,37 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/sveasoft/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/block_action]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/check_utilities [hunk 3]
```diff
@@ -0,0 +1,51 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/check_utilities]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/constants [hunk 4]
```diff
@@ -0,0 +1,16 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/constants]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/ip_forwarding [hunk 5]
```diff
@@ -0,0 +1,20 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/ip_forwarding]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/kernel_vars [hunk 6]
```diff
@@ -0,0 +1,29 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/kernel_vars]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/load_modules [hunk 7]
```diff
@@ -0,0 +1,43 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/load_modules]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/prolog_epilog_functions [hunk 8]
```diff
@@ -0,0 +1,28 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/prolog_epilog_functions]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/reset_iptables [hunk 9]
```diff
@@ -0,0 +1,49 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/reset_iptables]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/routing_functions [hunk 10]
```diff
@@ -0,0 +1,72 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/routing_functions]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/run_time_wrappers [hunk 11]
```diff
@@ -0,0 +1,70 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/run_time_wrappers]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/script_body_iptables_restore [hunk 12]
```diff
@@ -0,0 +1,42 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/script_body_iptables_restore]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/script_body_iptables_shell [hunk 13]
```diff
@@ -0,0 +1,25 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/script_body_iptables_shell]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/script_body_single_rule [hunk 14]
```diff
@@ -0,0 +1,32 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/script_body_single_rule]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/status_action [hunk 15]
```diff
@@ -0,0 +1,44 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/sveasoft/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/status_action]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/stop_action [hunk 16]
```diff
@@ -0,0 +1,34 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/sveasoft/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/stop_action]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/update_bonding [hunk 17]
```diff
@@ -0,0 +1,182 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/update_bonding]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/update_bridge [hunk 18]
```diff
@@ -0,0 +1,124 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/update_bridge]: `fwbuilder/configlets/linux317`

File: src/res/configlets/linux317/update_vlans [hunk 19]
```diff
@@ -0,0 +1,126 @@
+## -*- mode: shell-script; -*- 
+##
+## To be able to make changes to the part of configuration created
+## from this configlet you need to copy this file to the directory
+## fwbuilder/configlets/linux24/ in your home directory and modify it.
```
Comment (sirius) [inline src/res/configlets/linux317/update_vlans]: `fwbuilder/configlets/linux317`

**Comment by gaima8** [comment]
Hey Sirius,

Thanks for the review.
I've made the changes you requested.


Mike

**Pull request merged**